pkg/seccomp/default_linux.go

```diff
@@ -50,31 +50,210 @@ func DefaultProfile() *Seccomp {
 	syscalls := []*Syscall{
 		{
 			Names: []string{
+				"_sysctl",
+				"add_key",
+				"arc_gettls",
+				"arc_settls",
+				"arc_usr_cmpxchg",
+				"atomic_barrier",
+				"atomic_cmpxchg_32",
 				"bdflush",
+				"cachectl",
+				"clone2",
```

Comment:
Quite similar `clone` call is added to allow list below. Should `clone` be also moved to `EPERM` list if we are adding `clone2` here.

Our current allow and blocklist is bit different from `docker` though: https://docs.docker.com/engine/security/seccomp/#significant-syscalls-blocked-by-the-default-profile

Reply:
I agree, and added `clone` as well as `clone3` to the block list. I'd like to leave this decision up to the other maintainers since I'm not sure about the reasoning why we added `clone` to the allow list.

Reply:
Why would we want to block clone?

Reply:
Hm, looks like containerd is allowing `clone` but blocking `clone3`:
https://github.com/containerd/containerd/blob/b521429b67c0396fa177d83d6f4635e82fb733de/contrib/seccomp/seccomp_default.go#L656-L680

Reply:
It return ENOSYS on clone3 (see https://github.com/moby/moby/pull/42681) to ensure glibc will correctly fallback to using clone.

I really want @giuseppe's eyes on this PR.  Let's make sure to leave breadcrumbs in the commit message and code comments to not forget the reasoning.

Reply:
We want to allow clone so that we can run containers within containers without having to disable seccomp.
It would be so nice if there was a way to add and remove syscalls easily in podman.  But Humans will have a hard time figuring this out.

Reply:
Alright, I moved clone2 to the allowlist in a dedicated commit.